server/routes/stock-news.js

```diff
@@ -191,5 +191,9 @@ router.get('/allNewsInfo', function (req, res) {
     res.send(newsInfo)
 });
 
+router.post('/allNewsSources', function (req, res) {
```

Comment:
Please do not use the same endpoint
This has already been used in a get request